src/store/index.ts

```diff
@@ -0,0 +1,24 @@
+import { createStore, compose, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
+import rootReducer from './reducers';
+import rootSaga from './sagas';
+
+declare global {
+    interface Window {
+      __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
+    }
+}
+const sagaMiddleware = createSagaMiddleware();
+const initialState = {};
+
+const withDevTools = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
it should be activated only in the DEV environment!

Reply:
done